### PR TITLE
new icon: eclipse (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #1215: [ICON REQUEST]: Eclipse
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Eclipse

### Links and sources

https://iconape.com/eclipse-2-logo-icon-svg-png.html 

### Additional information

Thank you

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [ ] PR does not match another non-stale PR currently opened
- [ ] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [ ] PR's base is the `develop` branch.
- [ ] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [ ] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [ ] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1215**

## Link to prove your SVG is correct and up-to-date.

https://www.eclipse.org/ide/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -7195,5 +7195,24 @@
                 "alias": "plain"
             }
         ]
+    },
+    {
+        "name": "eclipse",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:7201]:
```suggestion
        "altnames": [
            "Eclipse IDE"
        ],
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Add alias and tags then we're good :)

Logos look fine

**Comment by kilian-paquier** [comment]
> Add alias and tags then we're good :)
> 
> Logos look fine

Hello @Snailedlt,

An alias between which version and which other version ? Because I made SVG's for the plain and SVG's for the original. Maybe the line then ?

**Comment by Snailedlt** [comment]
Ahh, no I meant altname, not alias. Sorry my bad :)

**Comment by kilian-paquier** [comment]
@Snailedlt Suggestions applied

**Review by Snailedlt — APPROVED** [review]

**Comment by Snailedlt** [comment]
@Panquesito7 Looks good to me

**Review by Snailedlt — APPROVED** [review]
LGTM!

**Pull request merged**